CHANGES/7826.bugfix

```diff
@@ -0,0 +1 @@
+Fix semver comparison
```

Comment:
Can this describe the sitaution more?

Reply:
Fix semver comparison for build numbers?

Reply:
Maybe:     `Fix version comparisons during sync and upload when comparing the same version with different build numbers.`